src/CreeDictionary/CreeDictionary/templatetags/relabelling.py

```diff
@@ -37,7 +40,7 @@ def relabel(context: Context, tags: Sequence[FSTTag], labels=None):
     else:
         label_setting = labels
 
-    relabeller = label_setting_to_relabeller[label_setting]
+    relabeller = label_setting_to_relabeller()[label_setting]
```

Comment:
If accepting the previous suggestion:

```suggestion
    relabeller = label_setting_to_relabeller(label_setting)
```